code/modules/mob/mob.dm

```diff
@@ -96,22 +96,21 @@
 // blind_message (optional) is what blind people will hear e.g. "You hear something!"
 
 /mob/visible_message(var/message, var/self_message, var/blind_message)
-	var/list/messageturfs = list()//List of turfs we broadcast to.
-	var/list/messagemobs = list()//List of living mobs nearby who can hear it, and distant ghosts who've chosen to hear it
-	for (var/turf in view(world.view, get_turf(src)))
-		messageturfs += turf
+	var/list/see = get_mobs_or_objects_in_view(world.view,src) | viewers(world.view,src)

```

Comment:
Nooope.

You're reverting all my changes to visible messages, i did this for a reason
What exactly is the problem you're trying to fix here?